spec/kicker.spec.js

```diff
@@ -221,9 +225,62 @@ describe('Kicker', function () {
             writerMock.verify()
           })
       })
+
+      after(function () {
+        processHostMock.restore()
+        writerMock.restore()
+      })
+    })
+
+    describe('when timeout is not set', function () {
```

Comment:
I'm a little bit confused by this test description. In line 258, it _does_ set a timeout. Is it just because that timeout is then suuuuper long?

Reply:
Sorry - confusing wording - it should read `changeWait` not `timeout` - under the hood this sets a timeout. Simplified this test and changed the wording so it should make more sense now!